src/firmware/mod.rs

```diff
@@ -313,9 +313,18 @@ pub struct SnpStatus {
     /// The platform's current state.
     pub state: State,
 
+    /// IsRmpInitiailzied
+    pub is_rmp_init: bool,
+
+    /// MaskChipId
+    pub mask_chip_id: bool,
+
     /// The number of valid guests supervised by this platform.
     pub guests: u32,
 
     /// The installed TCB version.
     pub tcb_version: u64,
+
+    /// reported TCB version
+    pub reported_tcb_version: u64,
```

Comment:
Since we're still not 1.0 does it make sense to change this into `tcb_version.installed` and `tcb_version.reported` instead?

Reply:
Would be interesting to know, what the difference is, and what to use.